snapcraft.yaml

```diff
@@ -19,44 +32,57 @@ slots:
 apps:
   evince:
     command: usr/bin/evince
-    extensions: [gnome-3-38]
+    extensions: [gnome]
     plugs:
       - avahi-observe
-      - cups-control
+      - cups
       - gsettings
       - home
       - network
       - removable-media
     desktop: usr/share/applications/org.gnome.Evince.desktop
+
   evince-previewer:
     command: usr/bin/evince-previewer
     plugs:
       - avahi-observe
-      - cups-control
+      - cups
       - gsettings
       - home
       - network
     desktop: usr/share/applications/org.gnome.Evince-previewer.desktop
+
   evince-thumbnailer:
     command: usr/bin/evince-thumbnailer
     plugs:
       - gsettings
```

Comment:
gsettings is part of the gnome extension, so it can be removed.

Reply:
But are you talking about the library, or the plug? Because, AFAIU, this is to give the snapped program access to the user's settings... does using the Gnome extension give that access automagically?

Reply:
I'm talking about the plug, at least [for gnome-3-38](https://github.com/snapcore/snapcraft/blob/main/snapcraft_legacy/internal/project_loader/_extensions/gnome_3_38.py#L117) this was the case.

Reply:
The plug is added automatically when using the gnome extension.  You can verify by removing it in the yaml then running "snapcraft expand-extensions" and you'll see gsettings plug is still in the resulting yaml.

Reply:
It complains about the "parse-info" statement, but if I remove it, you are right: gsettings is there. Removing.

Reply:
Done.